src/Scenario/Processor.php

```diff
@@ -64,19 +64,26 @@ public function execute(array $scenarios): void
             implode(', ', $scenarios),
             $this->packageManager->getPrettyInfo()
         ));
+        $steps = [];
 
         try {
             $this->mergedScenarios = $this->merger->merge($scenarios);
             $steps = $this->mergedScenarios['steps'];
-
+        } catch (Throwable $exception) {
+            $this->handleException(
+                $exception,
+                sprintf('Unhandled error: [%d] %s', $exception->getCode(), $exception->getMessage())
+            );
+        }
```

Comment:
Please add a newline to separate try/catch blocks

Reply:
done